app/_gateway_entities/route.md

```diff
@@ -4,57 +4,176 @@ content_type: reference
+
+Routes can also define rules that match requests to associated Services. Because of this, one Route can reference multiple endpoints. Once a Route is matched, {{site.base_gateway}} proxies the request to its associated Service. A basic Route should have a name, path or paths, and reference an existing Service.
+
+{% mermaid %}
+flowchart LR
+  A(API client)
+  B("`Route 
+  (/mock)`")
+  C("`Gateway Service
+  (example-service)`")
+  D(Upstream 
+  application)
+  
+  A <--requests
+  responses--> B
+  subgraph id1 ["`
+  **KONG GATEWAY**`"]
+    B <--requests
+    responses--> C
+  end
+  C <--requests
+  responses--> D
+
+  style id1 rx:10,ry:10
+  
+{% endmermaid %}
+
+## Route and Service interaction
+
+Routes, in conjunction with [Services](/gateway/entities/service/), let you expose your Services to applications with {{site.base_gateway}}. {{site.base_gateway}} abstracts the Service from the applications by using Routes. Since the application always uses the Route to make a request, changes to the Services, like versioning, don’t impact how applications make the request. Routes also allow the same Service to be used by multiple applications and apply different policies based on the Route used.
+
+For example, say you have an external application and an internal application that need to access the `example_service` Service, but the *external* application should be limited in how often it can query the Service to avoid a denial of service. If you apply a rate limit policy to the Service and the *internal* application calls it, the internal application is also limited. Routes can solve this problem.
+
+In this example, you can create two Routes to handle the two applications, say `/external` and `/internal`, and point both of them to `example_service`. 
+You can configure a policy to limit how often the `/external` Route is used. 
+When the external application tries to access the Service via {{site.base_gateway}} using `/external`, it's rate limited. 
+But when the internal application accesses the Service using {{site.base_gateway}} using `/internal`, the internal application isn't limited.
+
+The following diagram illustrates this example:
+
+{% mermaid %}
+flowchart LR
+  A(External application)
+  B("`Route (/external)`")
+  B2(Rate Limiting 
+  plugin)
+  C("`Service (example-service)`")
+  D(Upstream application)
+  E(Internal application)
+  F("`Route (/internal)`")
+
+  A --request--> B
+  E --request--> F
+
+  subgraph id1 ["`
+  **KONG GATEWAY**`"]
+    B --> B2 --"10 requests 
+    per minute"--> C
+    F ---> C
+  end
+
+  C --transformed 
+  and routed 
+  requests--> D
+
+  style id1 rx:10,ry:10
+
+{% endmermaid %}
+
+## Route use cases
+
+Common use cases for Routes:
+
+| You want to... | Then use... |
```

Comment:
```suggestion
| Use Case | Description |
```
This is a more dry title, but the old one doesn't fit when you're shortening the use case.